bigchaindb_driver/pool.py

```diff
@@ -46,9 +52,16 @@ def pick(self, connections):
                 :class:`~bigchaindb_driver.connection.Connection` instances.
 
         """
-        self.picked += 1
-        self.picked = self.picked % len(connections)
-        return connections[self.picked]
+
+        self.current_time_ms = datetime.now()
```

Comment:
`current_time_ms` can be a local variable, I guess it's safe to remove `self` (and remove it form the `__init__`).